conjur/data_object/conjurrc_data.py

```diff
@@ -29,11 +31,14 @@ def load_from_file(cls, conjurrc_path=DEFAULT_CONFIG_FILE):
         """
         Method that loads the conjurrc into an object
         """
-        with open(conjurrc_path, 'r') as conjurrc:
-            loaded_conjurrc = yaml_load(conjurrc, Loader=YamlLoader)
-            return ConjurrcData(loaded_conjurrc['conjur_url'],
-                                loaded_conjurrc['conjur_account'],
-                                loaded_conjurrc['cert_file'])
+        try:
+            with open(conjurrc_path, 'r') as conjurrc:
+                loaded_conjurrc = yaml_load(conjurrc, Loader=YamlLoader)
+                return ConjurrcData(loaded_conjurrc['conjur_url'],
+                                    loaded_conjurrc['conjur_account'],
+                                    loaded_conjurrc['cert_file'])
+        except KeyError:
+            raise InvalidConfigurationException
```

Comment:
@mbenita-Cyberark this is the most interesting part of the PR. This is where I catch if the keys don't exist